RELEASENOTES-1.4.docu

```diff
@@ -558,4 +558,15 @@
       in multiple inheritance: it allows any method that overrides multiple
       implementations from unrelated templates to individually reference and
       call the implementations it overrides.</add-note></build-id>
+  <build-id _6="next" _7="next"><add-note>Addressed a bug where DMLC did
+      not report errors when a non-existing template is instantiated within
+      an <tt>#if</tt> block. E.g., the following does not give an error:
+      <pre>
+      #if (true) { group g { is nonexisting_template; }}
+      </pre>
+      Our bugfix is opt-in, because an immediate bugfix would risk
```

Comment:
Still saying "our bugfix" here! Use "this (bug)fix" or "the (bug)fix" instead. Otherwise it's fine.

Reply:
done